x-pack/plugins/maps/public/classes/util/can_skip_fetch.test.ts

```diff
@@ -156,30 +154,30 @@ describe('canSkipSourceUpdate', () => {
         };
 
         const canSkipUpdate = await canSkipSourceUpdate({
-          source: queryAwareSourceMock,
+          source: (queryAwareSourceMock as unknown) as ISource,
           prevDataRequest,
-          nextMeta,
+          nextRequestMeta,
           extentAware: queryAwareSourceMock.isFilterByMapBounds(),
           getUpdateDueToTimeslice,
         });
 
         expect(canSkipUpdate).toBe(true);
       });
 
-      it('can not skip update when query is refreshed', async () => {
-        const nextMeta = {
+      it('Should not skip when force-refresh is on, regardless of applyGlobalQuery state', async () => {
```

Comment:
Nit - this test name captures churn in this PR. There is no link between applyGlobalQuery and isForceRefresh, so no need to mention this in the test name. How about just renaming to "Should not skip refresh update when applyForceRefresh is true"

Reply:
Can you add another test case testing "Should skip refresh update when applyForceRefresh is false"